### DISCUSSION TITLE
./gradlew core:zipRiruRelease failed

### DISCUSSION THREAD

**Discussion by nkbai** [LSPosed/LSPosed#1735] [Q&A]
Configuration on demand is an incubating feature.

> Configure project :core
WARNING:The option setting 'android.experimental.enableNewResourceShrinker.preciseShrinking=true' is experimental.
The current default is 'false'.
WARNING:The option setting 'android.enableAppCompileTimeRClass=true' is experimental.
The current default is 'false'.

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':core:prepareMagiskFilesRiruRelease'.
> A problem occurred configuring project ':daemon'.
   > Extension with name 'android' does not exist. Currently registered extension names: [ext]

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

**Comment by yujincheng08** [comment]
Why are u creating duplicate issues & discussions?
